README.md

```diff
@@ -79,7 +79,8 @@ When(a.ConfigMap)
 
 ## Prerequisites
 
-- [Node.js](https://nodejs.org/en/) v18.0.0+.
+- [Node.js](https://nodejs.org/en/) v18.0.0+ (even-numbered releases recommended).
```

Comment:
```suggestion
- [Node.js](https://nodejs.org/en/) v18.0.0+ (even-numbered releases only).
```
Slightly stronger verbiage maybe?  Since we've taken deps that only support even numbered releases AND we're not (at least, right now) prepared to do the work to pull them out / add odd-numbered support to them ourselves -- it'd be better to just declare ourselves even-only, straight-up.

Reply:
great point